README.md

```diff
@@ -1,33 +1,67 @@
 
-## Notes
+```
+post = Post.new
+post[:title] = "Title Goes Here"
+post[:description] = "Desc goes here..."
+post.save
+```
 
-Given a posts#new and a form.html.erb pointing to posts#create and passing data, how do we actually make a post and persist to our database.
+This syntax will let you manually create a new `Post` record with a `title` and `description` attribute. After running the `save` method in the console you will see output similar to the following:
 
-we know from console, we can most a post with Post.new and then setting attributes and saving it.
+```
+ (0.1ms)  begin transaction
+SQL (0.3ms)  INSERT INTO "posts" ("title", "description", "created_at", "updated_at") VALUES (?, ?, ?, ?)  [["title", "Title Goes Here"], ["description", "Desc goes here..."], ["created_at", "2015-11-23 22:26:43.799742"], ["updated_at", "2015-11-23 22:26:43.799742"]]
+ (1.2ms)  commit transaction
+=> true
 
-Let's copy that code to posts#create.
+```
 
-Instead of having hard coded data, where is our form data located? params.
+As you can see, the `save` method generates a SQL script that inserts a new record into the database, passing the `title` and `description` parameters into the statement and then returning the newly created `Post` object. At a high level this is what the `create` method in our controller will be doing.
 
-Let's use params to manualy assign the post data from the form to the instance of the post and save it.
+Opening up the `posts_controller.rb` file, let's do a few things to replicate the behavior we had in the console:
 
-After we save it, lets submit the form and use the console to see if the post was created.
+1. Create a new `Post` instance
 
-it was.
+2. Pass in the parameters from the form
 
-but we got a rails error. why? because at the end of the posts#create, we left no instructions and rails default took over and tried rendering create.html.erb. What do we want to happen at the end of a create? let's redirect to posts#show for that post
+3. Save the record
 
-redirect_to helper method in a controller.
+To build in this behavior initially you can copy and paste the code that we ran in the console. The only key difference is that now we want to pull in the form data and have that populate the `title` and `description` attributes, you can access each of the form elements by using the hash syntax to grab the elements from the `params` hash that is submitted with the form. The new code in the `create` method should look something like this:
 
-lets use post_path(@post) to redirect to the newly created post. And we're all good.
+```ruby
+def create
+  post = Post.new
+  post[:title] = params[:title]
+  post[:description] = params[:description]
+  post.save
+end
+```
+
+If you go to `/posts/new` and fill out the form and submit it, you'll get the error shown below, but this is ok, Rails is simply complaining that we don't have a view template since by default it's trying to render a `create` template that doesn't exist.
```

Comment:
Mention "a view template called create.html.erb". Just gotta keep reminding them of that auto-naming convention stuff. So much magic